src/cap32.cpp

```diff
@@ -1316,13 +1360,37 @@ int video_set_palette ()
       }
    } else {
       for (int n = 0; n < 32; n++) {
+
+         double *colours_green;
+
+         if (args.greenMode == 0 || args.greenMode == 1) {
+            // mode 0 and 1 -> caprice classic
+            colours_green = colours_green_classic;
+         } else {
+            // mode 2 and 3 -> libretro proposal
+            colours_green = colours_green_libretro;
+         }
+
          dword green = static_cast<dword>(colours_green[n] * (CPC.scr_intensity / 10.0) * 255);
          if (green > 255) {
             green = 255;
          }
+
+         dword blue = 0;
+
+         // add a tiny bit of blue for green mode 1 and 3
+         if (args.greenMode == 1 || args.greenMode == 3) {
```

Comment:
You could have 2 variables in the config: one for the green screen mode (classic / libretro) and one for the proportion of blue (the hardcoded 0.31 in your formula).